services/pseudo_controller/src/main/java/com/futurewei/alcor/pseudo_controller/pseudo_controller.java

```diff
@@ -0,0 +1,322 @@
+/*
+This is the code for the test controller, for testing the reactions between the Network Configuration manager and
+the ACA.
+
+Params:
+1. Number of ports to generate to each aca node
+2. IP of aca_node_one
+3. IP of aca_node_two
+4. IP of the GRPC call
+5. Port of the GRPC call
+6. User name of aca_nodes
+7. Password of aca_nodes
+*/
+package com.futurewei.alcor.pseudo_controller;
+import com.futurewei.alcor.schema.*;
+import com.jcraft.jsch.Channel;
+import com.jcraft.jsch.ChannelExec;
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.Session;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.stub.StreamObserver;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+
+public class pseudo_controller {
+
+    static String aca_node_one_ip = "ip_one";
+    static String aca_node_two_ip = "ip_two";
+    static String grpc_ip = "ip_three";
+    static int grpc_port = 123;
+    static String user_name = "root";
+    static String password = "abcdefg";
+    static int ports_to_generate_on_each_aca_node = 1;
+    static String vpc_id_1 = "2b08a5bc-b718-11ea-b3de-111111111112";
+    static String port_id_1 = "11111111-b718-11ea-b3de-111111111112";
+    static String port_id_2 = "13333333-b718-11ea-b3de-111111111114";
+    static String subnet_id_1 = "27330ae4-b718-11ea-b3df-111111111113";
+    public static void main(String[] args){
+        System.out.println("Start of the test controller");
+        if(args.length == 7){
+            System.out.println("User passed in params and we need to read them.");
+            ports_to_generate_on_each_aca_node = Integer.parseInt(args[0]);
+            aca_node_one_ip = args[1];
+            aca_node_two_ip = args[2];
+            grpc_ip = args[3];
+            grpc_port = Integer.parseInt(args[4]);
+            user_name = args[5];
+            password = args[6];
+
+        }
+
+        System.out.println("aca_node_one_ip: " + aca_node_one_ip + "\naca_node_two_ip: " + aca_node_two_ip + "\nuser name: "+user_name+"\npassword: "+password);
+//        execute_ssh_commands("docker run -itd --name test1 --net=none busybox sh", aca_node_one_ip, user_name, password);
```

Comment:
be sure to delete the docker containers at the end of the run:
docker rm test1 -f

Reply:
fised